openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -180,6 +180,15 @@ def get_response_with_refreshed_jwt_cookies(request, user):
     return response
 
 
+def update_jwt_cookies_in_response(request, response, user):
```

Comment:
Are you sure we don't want to just use `set_logged_in_cookies` to minimize code and potential sync issues between these cookies?

Reply:
Thanks for poininting it out. You are right about the sync issues. I am now using `set_logged_in_cookies` function.